include/Make/Install.make

```diff
@@ -122,7 +122,7 @@ $(DESTDIR)$(INST_DIR) $(DESTDIR)$(UNIX_BIN):
 	$(MAKE_DIR_CMD) $@
 
 $(STARTUP): $(ARCH_DISTDIR)/$(GRASS_NAME).tmp
-	sed -e 's#'@GISBASE@'#'$(INST_DIR)'#g' \
+	sed -e 's#'@SET_GISBASE@'#'$(INST_DIR)'#g' \
```

Comment:
My guess is that this is silently failing. It is a replace and the string being replaced is missing, so no replace happens. I don't know why it is not failing, but maybe there is some fallback in the code, I would have to check.

Reply:
It does work.

Reply:
Right, include/Make/Install.make is modified accordingly. I guess it is because of `set(SET_GISBASE ${GISBASE})` in `lib/init/CMakeLists.txt`. The name is strange, but this may need some changes anyway in the near future (thinking FHS).

Reply:
You may be right. I didn't test Autotools in this branch. I'm replacing it with `@GISBASE@`.

Reply:
`SET_GISBASE` is the variable that sets `@GISBASE@` in:

https://github.com/OSGeo/grass/blob/83e7fe7ce52ebd32312486ff15424b8d03106dc8/lib/init/grass.py#L95

That path should be the install path.

Autotools solves this with a hack:

https://github.com/OSGeo/grass/blob/83e7fe7ce52ebd32312486ff15424b8d03106dc8/include/Make/Install.make#L131-L133

which is not possible in cmake. The variable name change in lib/init/grass.py is needed for the cmake configuration generation works (and cascades into current Makefiles too).

Replacing `@SET_GISBASE@` with  `@GISBASE@ ` as proposed breaks build. Reverted with e704ce2.

This is tested with both autotools and cmake, with and without FHS.

Reply:
We can possibly rename `SET_GISBASE` to `GISBASE_INSTALL_PATH` for added clarification.

Reply:
-1 for hacks
+1 for readable names